server.py

```diff
@@ -1,16 +1,25 @@
 import socket
+from threading import Thread
 
-class Server:
+class Server(Thread):
     ''' Server allows the standing up of a server to listen for connections, echoing back any message it recieves '''
-    saddr = (socket.gethostname(), 9999) # create the server address using the local machine name
-    actionqueue = None                   # the ActionQueue the server is running alongside
 
-    def start():
+    def __init__(self, aq, addr=socket.gethostname(), port=9999):
```

Comment:
If nothing else, the port should be 5011 not 9999. Ideally we want addr to be `socket.gethostbyname(socket.gethostname())`
as the current version returns the name of the machine.